index-fr.md

```diff
@@ -10,6 +10,8 @@ modified: 2019-04-11
 <h2><span class="fa-stack"> <span class="fa fa-circle fa-stack-2x"></span> <span class="fa fa-font fa-stack-1x fa-inverse"></span> </span> Création de contenu</h2>
 <p>Avant de créer du contenu, les créateurs de contenu et les fournisseurs de contenu doivent comprendre ce qui suit et s'y conformer : </p>
 <dl>
+<dt><a href="accessibility/index-fr.html">Accessibility guideline</a></dt>
+<dd>Provides content authors and webmasters with an understanding of how to create and code content so that is accessible to all users.</dd>
```

Comment:
```suggestion
<dd lang="en">Provides content authors and webmasters with an understanding of how to create and code content so that is accessible to all users.</dd>
```